Modules/Package.swift

```diff
@@ -47,7 +47,7 @@ let package = Package(
         .package(url: "https://github.com/wordpress-mobile/FSInteractiveMap", from: "0.3.0"),
         .package(url: "https://github.com/wordpress-mobile/MediaEditor-iOS", branch: "task/spm-support"),
         .package(url: "https://github.com/wordpress-mobile/NSObject-SafeExpectations", from: "0.0.6"),
-        .package(url: "https://github.com/wordpress-mobile/NSURL-IDN", branch: "trunk"),
+        .package(url: "https://github.com/wordpress-mobile/NSURL-IDN", revision: "b34794c9a3f32312e1593d4a3d120572afa0d010"),
```

Comment:
This comes from https://github.com/wordpress-mobile/NSURL-IDN/pull/12

I think we should keep the package pointing to a revision instead than a branch. It's the closest thing to locking to a version when we don't want to got through the trouble of shipping one for this [obsolete package which we'll be able to remove once on iOS 17 and above](https://github.com/Automattic/apps-infra-toolbox/issues/110#issuecomment-2493038975)